### PR TITLE
Support grant privilege for MySQL 8

### PR DESCRIPTION
**Pull request by martin-walsh**

### CODE REVIEW

**Review by sagauhar — DISMISSED** [review]

**Review by sagauhar — APPROVED** [review]

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+steps:
+  - label: ':docker: Push Image'
+    command: ./ci-scripts/push-image.sh
+    env:
+      BINARY_OS: linux
+      BINARY_ARCH: amd64
+      IMAGE_REPOSITORY: 278521702583.dkr.ecr.us-west-2.amazonaws.com/iops/terraform-provider-mysql
+      IMAGE_TAG: "build-${BUILDKITE_BUILD_NUMBER}"
```
Comment (cameronattard) [inline .buildkite/pipeline.yml:8]: Will tagged releases get a proper version tag?

Comment (cameronattard) [reply .buildkite/pipeline.yml:8]: Nevermind - read the code below

Comment (sagauhar) [reply .buildkite/pipeline.yml:8]: yes...just like apps

Comment (sagauhar) [reply .buildkite/pipeline.yml:8]: logic is in the push-image.sh

**Pull request merged**